scripts/sysdig-probe-loader

```diff
@@ -283,16 +283,17 @@ load_bpf_probe() {
 ARCH=$(uname -m)
 KERNEL_RELEASE=$(uname -r)
 SCRIPT_NAME=$(basename "${0}")
-SYSDIG_PROBE_URL=${SYSDIG_PROBE_URL:-https://s3.amazonaws.com/download.draios.com}
+CONFIG_FILE="/etc/sysdig/${SCRIPT_NAME}"
+if [[ -f "${CONFIG_FILE}" ]]; then
+        source "${CONFIG_FILE}"
```

Comment:
I don't think we want to directly source an arbitrary file into the probe loader. I understand that the intent is to override certain config options, but that mechanism is very generic and could also override the functions like `load_kernel_probe()` in the script itself. This could cause the script to do virtually anything, in this case as root.

Reply:
@mstemm may I remind you that this script basically downloads a binary blob from the internet and loads it as a kernel module... That seems to qualify as "virtually anything" 😀

I could source the file at the top of the script, so it can not override the functions, if that is the worry...